README.md

```diff
@@ -3,10 +3,10 @@
 ## Install
 
 ```fish
-$ git clone git://github.com/evanlucas/fish-kubectl-completions
 $ mkdir -p ~/.config/fish/completions
-$ cd fish-kubectl-completions
-$ ln -s (pwd)/kubectl.fish ~/.config/fish/completions/
+$ cd ~/.config/fish
+$ git clone https://github.com/evanlucas/fish-kubectl-completions
+$ ln -s ../fish-kubectl-completions/kubectl.fish completions/
```

Comment:
I think this would need to be `ln -s ./fish-kubectl-completions/kubectl.fish completions/`. Otherwise, it will assume that the cloned directory is a level up.

Reply:
No, it's not like that.

For `ln -s`, the first argument is saved into the symlink *verbatim*, then it's resolved in the context of that symlink's location. `~/.config/fish/completions/kubectl.fish` linking to `../fish-kubectl-completions/kubectl.fish` will resolve to `~/.config/fish/fish-kubectl-completions/kubectl.fish`

Reply:
ah yea, good point.